packages/runner/injection/cross-origin.js

```diff
@@ -40,6 +41,8 @@ const findCypress = () => {
 
 const Cypress = findCypress()
 
+patchDocumentCookie(Cypress)
```

Comment:
would it be valuable to patch this in a non-cy.origin use case? It _seems_ like it might be if it makes document.cookie behave more like as if the AUT is top.

Reply:
It's only an issue if the top origin doesn't match the AUT origin, so it's not necessary in other cases.